erizo/src/erizo/MediaStream.cpp

```diff
@@ -677,6 +682,52 @@ void MediaStream::getJSONStats(std::function<void(std::string)> callback) {
   });
 }
 
+void MediaStream::changeDeliverExtensionId(DataPacket *dp, packetType type) {
+  RtpHeader* h = reinterpret_cast<RtpHeader*>(dp->data);
+  RtcpHeader *chead = reinterpret_cast<RtcpHeader*>(dp->data);
+  if (!chead->isRtcp()) {
+    // Extension Id to external
+    if (h->getExtension()) {
+      std::array<RTPExtensions, 15> extMap;
+      RtpExtensionProcessor& ext_processor = getRtpExtensionProcessor();
+      switch (type) {
+        case VIDEO_PACKET:
+          extMap = ext_processor.getVideoExtensionMap();
+          break;
+        case AUDIO_PACKET:
+          extMap = ext_processor.getAudioExtensionMap();
+          break;
+        default:
+          ELOG_WARN("%s Won't process RTP extensions for unknown type packets", toLog());
+          return;
+          break;
+      }
+      uint16_t totalExtLength = h->getExtLength();
+      if (h->getExtId() == 0xBEDE) {  // One-Byte Header
+        char* extBuffer = (char*)&h->extensions;  // NOLINT
+        uint8_t extByte = 0;
+        uint16_t currentPlace = 1;
+        uint8_t extId = 0;
+        uint8_t extLength = 0;
+        while (currentPlace < (totalExtLength*4)) {
+          extByte = (uint8_t)(*extBuffer);
+          extId = extByte >> 4;
+          extLength = extByte & 0x0F;
+          // extId == 0 should never happen, see https://tools.ietf.org/html/rfc5285#section-4.2
+          for (int i = 1; i < 15; i++) {
```

Comment:
This is what I'm talking about. 
```suggestion
if (extId != 0) {
          for (int i = 1; i < 15; i++) {
            if (extMap.at(i) == extId) {
              extBuffer[0] = (extBuffer[0] | 0xF0) & (i << 4 | 0x0F);
            }
          }
         }
          extBuffer = extBuffer + extLength + 2;
          currentPlace = currentPlace + extLength + 2;         
```

Reply:
@lodoyun would you mind fixing the coding error over building

Reply:
@lodoyun I just ignore ```extId==0``` at for (int **i = **1****; i < 15; i++)

Reply:
The linter is failing because I did not respect the indentation, I probably shouldn't have used the suggestions feature. Just take the idea and use your own code.